packages/synapse-react-client/src/components/AclEditor/ResourceAccessItem.tsx

```diff
@@ -35,21 +42,28 @@ export const ResourceAccessItem: React.FunctionComponent<
           <UserOrTeamBadge principalId={resourceAccess.principalId} />
         </Typography>
         <Stack direction="row" gap="10px" alignItems="center" width="225px">
-          <PermissionLevelMenu
-            currentAccessType={resourceAccess.accessType}
-            availablePermissionLevels={availablePermissionLevels}
-            onChange={onChange}
-          />
-          <IconSvgButton
-            aria-label={REMOVE_BUTTON_LABEL}
-            onClick={() => onRemove()}
-            icon="delete"
-            sx={{
-              '&:hover': {
-                color: 'error.main',
-              },
-            }}
-          />
+          {!isInEditMode && (
+            <ReadOnlyPermissionLevel accessType={resourceAccess.accessType} />
+          )}
+          {isInEditMode && (
+            <>
+              <PermissionLevelMenu
+                currentAccessType={resourceAccess.accessType}
+                availablePermissionLevels={availablePermissionLevels}
+                onChange={onChange}
+              />
+              <IconSvgButton
+                aria-label={REMOVE_BUTTON_LABEL}
+                onClick={() => onRemove()}
+                icon="delete"
+                sx={{
+                  '&:hover': {
+                    color: 'error.main',
+                  },
+                }}
+              />
+            </>
+          )}
```

Comment:
If not in edit mode, just render the permission level text (no menu or delete button)